ST_structs.h

```diff
@@ -1,333 +1,721 @@
+         pseed;
+      /** \brief relsize from the previous year, used for annual establishment. 
+       * \sa rgroup_Establish() */
+  RealF lastyear_relsize,
+      /** \brief Amount of superfluous growth from extra resources. */
+        extragrowth,
+      /** \brief Probability that this species received seeds this year. */
+	      received_prob,
+      /** \brief Alpha parameter for random number draw from beta distribution in establishment of annual species. 
+       * \sa _add_annuals() */
+        alpha,
+      /** \brief Beta parameter for random number draw from beta distribution in establishment of annual species.
+       * \sa _add_annuals() */
+        beta;
+      /** \brief Variance parameter of the beta distribution for establishment. */
+  float var;
+      /** \brief Head of a doubly-linked list of all individuals of this species. 
+       * \sa indiv_st*/
+  struct indiv_st *IndvHead;
+      /** \brief Seed dispersal only- whether to allow growth for the current year.
+       * \sa ST_seedDispersal.c */
+  Bool allow_growth,
+      /** \brief Whether seeds where produced/received and germinated this year. 
+       * \sa ST_seedDispersal.c */
+	     sd_sgerm;
 
   /**** Quantities that DO NOT change during model runs *****/
 
-    /* 4-letter code (plus \0) for genus and species*/
+      /** \brief 4-letter code (plus \0) for genus and species. */
   char* name;
-  IntUS max_age,         /* max age of mature plant, also flag for annual */
-        viable_yrs,      /* annuals: max years of viability of seeds */
-        max_seed_estab,  /* max seedlings that can estab in 1 yr*/
-        max_vegunits,    /* max vegetative regrowth units (eg tillers)*/
-        max_slow,        /* years slow growth allowed before mortality */
-        sp_num,          /* index number of this species */
-        res_grp;         /* this sp. belongs to this res_grp*/
-  RealF max_rate,      /* intrin_rate * proportion*/
+      /** \brief Max age of mature plant. Also used to determine annual species. */
+  IntUS max_age,
+      /** \brief Annuals: max years of viability of seeds. */
+        viable_yrs,
+      /** \brief Max seedlings that can establish in 1 year. */
+        max_seed_estab,
+      /** \brief Max vegetative regrowth units, for example tillers. */
+        max_vegunits,
+      /** \brief Years slow growth is allowed before mortality. */
+        max_slow,
+      /** \brief Index of this species in the Species variable.
+       * \sa Species */
+        sp_num,
+      /** \brief Resource group that this species belongs to. 
+       * \sa RGroup */
+        res_grp;
+      /** \brief Max growth rate. Defined as intrin_rate * proportion. */
+  RealF max_rate,
+      /** \brief Intrinsic growth rate. */
         intrin_rate,
+      /** \brief Starting size of a new individual of this species, relative to an adult plant. */
         relseedlingsize,
+      /** \brief Starting biomass in grams of a new individual of this species. */
```

Comment:
Change to: 

Seedling biomass in grams of an individual of this species

Reply:
Addressed by commit 984bd13.